src/types.h

```diff
@@ -473,6 +473,15 @@ namespace librealsense
             }
         }
 
+        void reload() const
```

Comment:
Please explain why this method is needed.

Reply:
The difference with reset() is that unique_ptr remains allocated, but it is enforced to reinitialize internally -e.g. invoke the assignment lambda.
It is similar as if the "lazy" hasn't been activated yet